config/connectSlack.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console */
+const { App } = require("@slack/bolt");
+
+const modalTemplate = require("./modal.json");
+
+const app = new App({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  token: process.env.SLACK_BOT_TOKEN,
+  appToken: process.env.SLACK_APP_TOKEN,
+  socketMode: true,
+});
+
+(async () => {
+  await app.start(process.env.PORT || 3000);
+
+  console.log("⚡️ Bolt app is running!");
+})();
+
+(function () {
+  const channelList = {};
+
+  app.command("/code", async ({ ack, body, client, logger }) => {
+    const { channel_id } = body;
+
+    await ack();
+
+    try {
+      const result = await client.views.open({
+        trigger_id: body.trigger_id,
```

Comment:
여기서도 마찬가지로 구조 분해 할당으로 받도록 수정 부탁드립니다!

```suggestion
        { trigger_id: triggerId } = body;
```